build/stf-run-ci/tasks/pre-clean.yml

```diff
@@ -131,7 +130,10 @@
       apiVersion: project.openshift.io/v1
       kind: Project
       metadata:
-        name: openshift-cert-manager-operator
+        name: "{{ item }}"
+  loop:
+    - openshift-cert-manager-operator
+    - cert-manager-operator
```

Comment:
tech-preview channel vs stable-v1 channel uses different namespaces, so this covers both instances.